### PR TITLE
doc: note that `nix eval` is eager

### PR DESCRIPTION
**Pull request by fricklerhandwerk**
# Motivation
Got bitten by it.

# Context
Boyscout rule.

# Priorities and Process

Add :+1: to [pull requests you find important](https://github.com/NixOS/nix/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc).

The Nix maintainer team uses a [GitHub project board](https://github.com/orgs/NixOS/projects/19) to [schedule and track reviews](https://github.com/NixOS/nix/tree/master/maintainers#project-board-protocol).

### CODE REVIEW

File: src/nix/eval.md [hunk 1]
```diff
@@ -50,8 +50,7 @@ R""(
 
 # Description
 
-This command evaluates the given Nix expression and prints the
-result on standard output.
+This command eagerly evaluates the given Nix expression and prints the result on standard output.
```
Comment (roberth) [inline src/nix/eval.md:53]: Evaluation is never eager or strict. It always happens the same way.
The difference is in how evaluation is used; not intrinsic to evaluation itself.

```suggestion
This command also evaluates any nested attribute values and list items, and prints the result on standard output.
```

**Review by roberth — CHANGES_REQUESTED** [review]
Important info, thanks!

File: src/nix/eval.md [hunk 2]
```diff
@@ -50,8 +50,7 @@ R""(
 
 # Description
 
-This command evaluates the given Nix expression and prints the
-result on standard output.
+This command also evaluates any nested attribute values and list items, and prints the result on standard output.
```
Comment (fricklerhandwerk) [inline src/nix/eval.md:53]:
```suggestion
This command evaluates the given Nix expression, and prints the result on standard output.

It also evaluates any nested attribute values and list items.
```

**Review by roberth — APPROVED** [review]
I'd say it's a bit more connected, but this is good enough.

**Pull request merged**